cli/local/docker_spec.go

```diff
@@ -349,7 +400,25 @@ func deployTensorFlowContainers(api *spec.API, awsClient *aws.Client) error {
 
 	err = docker.MustDockerClient().ContainerStart(context.Background(), containerCreateRequest.ID, dockertypes.ContainerStartOptions{})
 	if err != nil {
-		return errors.Wrap(err, api.Identify())
+		if api.Compute.GPU > 0 && strings.Contains(err.Error(), "could not select device driver \"\" with capabilities: [[gpu]]") {
```

Comment:
Would it be easy to move this code block to a helper function to be used by each predictor?